lib/components/cards/VeggieImage.dart

```diff
@@ -20,13 +21,14 @@ class VeggieImage extends StatelessWidget {
     };
 
     return Container(
-        width: width,
-        height: height,
-        decoration: $styles.containerStyles.rounded,
-        child: ClipRRect(
-          borderRadius: $styles.borderStyle.roundedBorderRadius,
-          child: Image(
-              fit: BoxFit.cover, image: AssetImage(veggie.imageAssetPath)),
-        ));
+      width: width,
+      height: height,
+      decoration: $styles.containerStyles.rounded(
+        image: DecorationImage(
```

Comment:
IIRC - I think the ClipRRect was an artifact of the Image corners not respecting the Container decoration's rounded corner or something like that. But now knowing the decoration can take an image is a big plus!